quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/domain/DashboardRepository.java

```diff
@@ -1,13 +1,14 @@
-package com.github.reflectoring.infiniboard.quartermaster.dashboard;
+package com.github.reflectoring.infiniboard.quartermaster.dashboard.domain;
 
 import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Repository;
 
-import com.github.reflectoring.infiniboard.quartermaster.widget.WidgetConfigService;
+import com.github.reflectoring.infiniboard.quartermaster.widget.domain.WidgetConfigService;
 
 @Repository
 public class DashboardRepository {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)